src/modules/formBuilder/components/FormBuilderPalette.tsx

```diff
@@ -119,22 +119,21 @@ interface FormBuilderPaletteType {
 const FormBuilderPalette: React.FC<FormBuilderPaletteType> = ({
   onItemClick,
 }) => {
+  const top = STICKY_BAR_HEIGHT + CONTEXT_HEADER_HEIGHT;
+
   return (
-    <Drawer
-      variant='persistent'
-      open={true}
+    <Paper
```

Comment:
This didn't actually need to be a Drawer -- I simplified it to just Paper in the course of troubleshooting css, and ended up liking the simplification, so I'm proposing to keep it.